Library/Formula/proj.rb

```diff
@@ -1,10 +1,10 @@
 require 'formula'
 
 class Proj < Formula
-  homepage 'http://trac.osgeo.org/proj/'
-  url 'http://download.osgeo.org/proj/proj-4.8.0.tar.gz'
-  sha1 '5c8d6769a791c390c873fef92134bf20bb20e82a'
-  head "http://svn.osgeo.org/metacrs/proj/trunk/proj/"
+  homepage 'https://trac.osgeo.org/proj/'
+  url 'http://download.osgeo.org/proj/proj-4.9.1.tar.gz'
+  sha1 '0bc63a41f1bdcff600d076c056f796007abf3f2f'
+  head "https://svn.osgeo.org/metacrs/proj/trunk/proj/"
```

Comment:
Sadly the system ` svn ` doesn't support this, and we can't presume everyone has Homebrew's ` svn ` installed.

```
/usr/bin/svn checkout https://svn.osgeo.org/metacrs/proj/trunk/proj/
Error validating server certificate for 'https://svn.osgeo.org:443':
 - The certificate is not issued by a trusted authority. Use the
   fingerprint to validate the certificate manually!
Certificate information:
 - Hostname: *.osgeo.org
 - Valid: from Mon, 02 May 2011 00:00:00 GMT until Sun, 01 May 2016 23:59:59 GMT
 - Issuer: COMODO CA Limited, Salford, Greater Manchester, GB
 - Fingerprint: 72:2d:b7:e7:50:6d:84:73:ed:84:c9:98:0d:0f:91:58:28:a1:28:1b
(R)eject, accept (t)emporarily or accept (p)ermanently?
```